### PR TITLE
Bug fixes, bot process update

### PR DESCRIPTION
**Pull request by Markolie**
Changes:
1) Updates the bot controller so it functions similarly to the objects controller for maximum efficiency.
2) Removes the default examine from attack_ghost (single click). Ghosts can still examine with shift+click. With the new clicking system, you'd examine something twice when you jumped anywhere.
3) IPC's can now pick their frame (well, head frame) color by selecting a hair color.
4) When you take off surgical caps, you no longer lose your hair.
5) Fixes https://github.com/ParadiseSS13/Paradise/issues/317. Bit of a brute force solution, but it works.
6) Fixes https://github.com/ParadiseSS13/Paradise/issues/387.
7) Fixes https://github.com/ParadiseSS13/Paradise/issues/393.
8) Fixes https://github.com/ParadiseSS13/Paradise/issues/395.
9) Fixes mutetoxin.
10) Fixes the binary encryption key (and updates our headset code).
11) Rebalances the nuke ops skin color so it's a bit more balanced.
12) Fixes drone talk.

### CODE REVIEW

**Comment by Fox-McCloud** [comment]
I'd strongly recommend removing the encryption key from keyslot 1, by default; encryption keys literally do nothing; only departmental keys do anything.

File: paradise.dme [hunk 1]
```diff
@@ -189,7 +189,7 @@
 #include "code\controllers\Processes\obj.dm"
 #include "code\controllers\Processes\pipenet.dm"
 #include "code\controllers\Processes\powernet.dm"
-#include "code\controllers\Processes\shuttle.dm"
+#include "code\controllers\Processes\Shuttle.dm"
```
Comment (Fox-McCloud) [inline paradise.dme]: Meant to be capitalized?

File: code/game/objects/items/devices/radio/headset.dm [hunk 2]
```diff
@@ -15,13 +15,38 @@
 	var/obj/item/device/encryptionkey/keyslot2 = null
 	maxf = 1489
 
+	var/ks1type = /obj/item/device/encryptionkey
```
Comment (Fox-McCloud) [inline code/game/objects/items/devices/radio/headset.dm]: zeh encryption keys; zeh do nussing!

**Comment by fj45** [comment]
11) Rebalances the nuke ops skin color so it's a bit more balanced.

? Were people turning themselves the colour of floortiles or something?

**Comment by Markolie** [comment]
No, they were black 80% of the time. It should be 50-50 now.

**Pull request merged**